src/templates/gh_link.vue

```diff
@@ -0,0 +1,17 @@
+<!-- A version of http://tholman.com/github-corners/ with Vue support. -->
```

Comment:
License header is missing.

Reply:
Credit https://github.com/tholman/github-corners (under MIT license) instead of the site.